groups/clipping-shapes-masking.yml

```diff
@@ -0,0 +1,2 @@
+name: Clipping, shapes and masking
+parent: css
```

Comment:
The potential issue with this group is that it now contains features that apply to both CSS and SVG.
Your features make sense to me, and I'm fine with making them apply to both languages, if others are. But in that case, we need to make this group belong to both CSS and SVG (which is not possible today).

Reply:
Good point- I've been thinking we will need to address the overlap between CSS and SVG more thoroughly at some point. Allowing for multiple parents for a group would be complex, so I'll remove the parent group here for now.